config/default-settings.json

```diff
@@ -2,7 +2,7 @@
   "motd": "A Minecraft Server \nRunning flying-squid",
   "port": 25565,
   "max-players": 10,
-  "online-mode": true,
+  "online-mode": false,
```

Comment:
Why this change?

Reply:
Oops... Sorry about it... I forgot that I've changed this file... It was done for the testing purpose and actually is not intended to go in the repository. If you wish it to be like it was in the repository before you can either change it yourself or I will fix it in the next commit